zignal/audio.py

```diff
@@ -229,7 +229,7 @@ def _fade(self, millisec, direction):
         fade_seconds = millisec/1000
         assert self.duration > fade_seconds, "fade cannot be longer than the length of the audio"
 
-        sample_count = np.ceil(fade_seconds*self.fs)
+        sample_count = int(np.ceil(fade_seconds*self.fs))
```

Comment:
I agree with this change but I cannot reproduce the error. What version of numpy are you using? I am using numpy 1.17.0 and I do not see this error.

Reply:
I'm using numpy 1.18.1. Non-integer num for linspace was deprecated in 1.12 and removed in 1.18.0 ([reference](https://numpy.org/devdocs/release/1.18.0-notes.html#expired-deprecations))

Reply:
Ok, good find. I'll merge this, thanks!